portfolio/src/main/webapp/index.html

```diff
@@ -11,6 +11,28 @@
       <h1>Amy's Portfolio</h1>
       <p>Hi! I'm Amy and welcome to my portfolio!</p>
       <p>I'm a rising junior at UC San Diego studying Data Science!</p>
+      <h2>Background</h2>
+      <p>I am a four-time instructional assistant for the data structures 
```

Comment:
Is "four" a typo?

Reply:
I don't think so. I meant four-time as in I tutored for four iterations of this course. I'll remove it anyways because I think it's unnecessary.